language/move-compiler/src/typing/expand.rs

```diff
@@ -280,6 +295,7 @@ fn lvalue(context: &mut Context, b: &mut T::LValue) {
         L::Ignore => (),
         L::Var(_, ty) => {
             type_(context, ty);
+            core::check_non_fun(context, ty.as_ref())
```

Comment:
I think we will need something better here, because this setup (with skipping the arg on LocalCall) won't allow for nested macro usages. But maybe we don't want that anyway (at least right away?)

Reply:
Not sure why nesting should not work, and what we are skipping. I added a test case for nested macro calls.